cmd/crictl/image.go

```diff
@@ -91,6 +91,15 @@ var pullImageCommand = &cli.Command{
 			EnvVars: []string{"CRICTL_PULL_TIMEOUT"},
 		},
 	},
+	Subcommands: []*cli.Command{{
+		Name:      "jsonschema",
+		Aliases:   []string{"js"},
+		Usage:     "Display the JSON schema for the pod-config.json",
```

Comment:
why this for the image subcommand?

Reply:
pedantically: seems like a bleeding of the runtime service into the image client (not like they're actually separated)

Reply:
It's added here because of `crictl pull --pod-config pod-config.[json|yam]`.

Reply:
I am not sure I understand the logic of using it in `pull`. What is the output of a command will be?

Reply:
@SergeyKanzhelev the sandbox config provides additional context during the image pull. For example CRI-O uses it to separate pulls into dedicated cgroups: 

https://github.com/cri-o/cri-o/blob/88fe8766ede790f465e1e86b0eccba0818736b32/server/image_pull.go#L47